### ISSUE TITLE
only one track result can show in the leaderboard

### ISSUE THREAD

**Issue by aisxt** [emotion-analysis-project/SemEval2025-Task11#8]
I subimitted eng both track a and b sperately, but only one track result can show in the leaderboard. How can I fix it

**Comment by nedjmaou** [comment]
I believe we had a problem with CodaBench, tagging @shmuhammadd just in case.

**Comment by shmuhammadd** [comment]
@aisxt all is working now. Let us know if you are still having the problem.

**Comment by aisxt** [comment]
I still have the problem. I submitted two subtasks, English Dev B and Indonesian Dev C, at the same time, but on the Results interface, only English Dev B has a score of 0.631, while the Indonesian Dev C subtask shows N/A. On the My Submissions interface, I can also see that the Indonesian Dev C subtask has a score of 0.42.
![image](https://github.com/user-attachments/assets/35398eb8-6d13-4a32-b190-d9a4154937fe)
![image](https://github.com/user-attachments/assets/c7fc0616-5214-4929-9711-03525c1643b6)

**Comment by shmuhammadd** [comment]
> I still have the problem. I submitted two subtasks, English Dev B and Indonesian Dev C, at the same time, but on the Results interface, only English Dev B has a score of 0.631, while the Indonesian Dev C subtask shows N/A. On the My Submissions interface, I can also see that the Indonesian Dev C subtask has a score of 0.42. ![image](https://private-user-images.githubusercontent.com/34327247/399094666-35398eb8-6d13-4a32-b190-d9a4154937fe.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzU0MDI0MjksIm5iZiI6MTczNTQwMjEyOSwicGF0aCI6Ii8zNDMyNzI0Ny8zOTkwOTQ2NjYtMzUzOThlYjgtNmQxMy00YTMyLWIxOTAtZDlhNDE1NDkzN2ZlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjI4VDE2MDg0OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZkOWI0YjZiNTUyMWJhN2QwYTQ5MjgyODg0NDFmOGJhY2ViODZmNjcxN2NhMDNiNTA3YWFjNzcwNTIyYjNiMDkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.NQ4Fxorv_D-S2QHdHqB16v7ehXg8tVva6BoB6ViIq-0) ![image](https://private-user-images.githubusercontent.com/34327247/399095090-c7fc0616-5214-4929-9711-03525c1643b6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzU0MDI0MjksIm5iZiI6MTczNTQwMjEyOSwicGF0aCI6Ii8zNDMyNzI0Ny8zOTkwOTUwOTAtYzdmYzA2MTYtNTIxNC00OTI5LTk3MTEtMDM1MjVjMTY0M2I2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjI4VDE2MDg0OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWUyMTJmNzE0Y2ZmMTFhODQ1NjAwMTFjZGVlNmFmMzkxMjk2MzAwYmMwMjhmMTRhYmU5M2I0ZTY2MmJiMThiMWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.HLXy7ovnkSNf4Kh163yQkLDq3SM_xvVuZ96KmRheJ2g)

Can you submit them individually not at the same time?

**Comment by shmuhammadd** [comment]
@aisxt can you try submitting individually not at once ?

**Comment by aisxt** [comment]
Yes, I can, but if I have completed multiple tasks and want to show them on the results interface at the same time, what should I do? When I submit only one subtask each time, even if the subtasks I submit are different each time, the scores displayed on the results interface will be overwritten, and only one subtask result will always be displayed on the results interface. Can you help me solve this problem? Thank you very much

**Comment by shmuhammadd** [comment]
> Yes, I can, but if I have completed multiple tasks and want to show them on the results interface at the same time, what should I do? When I submit only one subtask each time, even if the subtasks I submit are different each time, the scores displayed on the results interface will be overwritten, and only one subtask result will always be displayed on the results interface. Can you help me solve this problem? Thank you very much

@aisxt I got it. There is an issue with Codabench. We will release task update on Monday. We will explain how this can be solved.

**Comment by shmuhammadd** [comment]
@aisxt check the task update and the updated submission guide. We explained how to solve this issue.
